java/values.yaml

```diff
@@ -45,3 +45,19 @@ global:
 prometheus:
   enabled: false
   path: /prometheus
+
+smoke-tests:
+  enabled: false
+  backoffLimit: 2
+
+smoke-tests-cron:
+  enabled: false
+  backoffLimit: 2
```

Comment:
we might need to set Kind as its defaulted to Job , but its driven from globals . So, we need to handle it better .

Reply:
That's a good point. The global job kind might be a problem.